master.tex

```diff
@@ -1,19 +1,20 @@
-\documentclass[
-version=last,toc=bib,toc=graduated,toc=index,toc=listof,fontsize=9pt,openany]{scrbook}
+\documentclass[version=last,toc=bib,toc=graduated,toc=index,toc=listof,fontsize=9pt,openany,twoside]{scrbook}
+%\documentclass[twoside]{article}
 %\pdfminorversion=4
 \usepackage{polyglossia}
 \setdefaultlanguage{english}
 
-\setmainfont{Source Sans Pro}
-\setsansfont{Source Sans Pro}
+\setmainfont{Barlow}
+\setsansfont{Barlow}
+\newfontfamily\numeral{d-din}
```

Comment:
No changes to the font please. Barlow is a too high font. I choose Source Sans Pro intentionally last year.

DIN font is no font for typesetting texts. It's a font for signs but we aren't producing destination signs for roads here.

Reply:
I think it is better to use Barlow/D-Din fonts consistently as it was originally proposed with our logo and design style guidelines. We also use Barlow on the Website. I believe it is well readable. The D-Din font is according to it only used for the year number "2019" in the context of "State of the Map 2019".